tools/refract-filter.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import argparse
+import sys
+import yaml
```

Comment:
This is a non-standard library import so it should be at the bottom of the imports as per PEP8.

http://pep8.org/#imports